app/discord_client.py

```diff
@@ -576,6 +577,15 @@ def check(reaction, user):
                     )
                 await message.delete()
         return
+    if txt.startswith("</suggest-topic"):
```

Comment:
Discord is deprecating message based commands btw, so this should at least be a slash command. See example on line 633.